_guide_topics/rhinocommon/plugin_installers_windows.md

```diff
@@ -28,35 +28,39 @@ Imagine you have a more complex plugin and want to support multiple versions of
 
 - Install an older version of the plugin for Rhino 4.0
 - Install a new version of the plugin for 32-bit Rhino 5
-- Install a plugin for 64-bit Rhino 5
-- Include a *MyToolbar.rui* custom toolbar file
-
-This is possible.  You need to ...
-
-1. *Organize the files*:  Create a "Installer Image" folder (In this example, the folder is the name of the product: Marmoset.)  This folder will contain only the files you want to install on the user's system...
-
-    - *Marmoset*\\
-        - *Rhino 4.0*\\
-            - *Marmoset.rhp*
-        - *Rhino 5.0*\\
-            - *x86*\\
-                - *Marmoset.rhp*
-            - *x64*\\
-                - *Marmoset.rhp*
-        - *Common*\\
-            - *Marmoset.rui*
-            - *Marmoset.chm*
-
-1. *Create the RHI file*: Copy the appropriate files into the folders.  Note that all three versions of the plugin can have the same name, so long as they are in different folders.
-1. Compress the three files using zip compression (currently other compression algorithms are not supported).
+- Install another version of the plugin for 64-bit Rhino 5
+- Include a custom toolbar file (e.g. *MyToolbar.rui*)
+
+This is possible.  You need to...
+
+1. Create a "Installer Image" folder (In this example, the folder is the name of the product: Marmoset.)  This folder will contain only the files you want to install on the user's system.
+
+        Marmoset/
+        ├── Rhino 4.0/
```

Comment:
It's technically not necessary for the folders to have any specific naming. Rhiexec will find compatible plug-ins and register them anywhere. I don't know if this detail is relevant or noise.